plasmapy/diagnostics/langmuir.py

```diff
@@ -1,153 +1,1073 @@
-"""
-Defines the Langmuir analysis module as part of the diagnostics package.
-"""
+"""Defines the Langmuir analysis module as part of the diagnostics package."""
 
-import numpy as np
+import plasmapy.constants as const
 import astropy.units as u
+import copy
+
+from plasmapy import utils
+
+from astropy.visualization import quantity_support
+import numpy as np
+import matplotlib.pyplot as plt
+
 
+def fit_func_lin(x, x0, y0, c0):
```

Comment:
This would probably work better as `_fit_func_lin` with the underscore in front to make this semi private!